lightning/src/ln/channelmanager.rs

```diff
@@ -786,20 +786,60 @@ macro_rules! maybe_break_monitor_err {
 }
 
 macro_rules! handle_chan_restoration_locked {
```

Comment:
Hmm, yea, it does refer to the `channel_state` lock, I guess I'm just afraid of getting the locking wrong in the future because we've had awkward locking incorrectness bugs in the past. If you feel strongly I can drop it.

Reply:
I don't feel strongly, resolving!